library/globals.inc.php

```diff
@@ -579,6 +579,13 @@ function gblTimeZones()
             xl('Are appointment category names to be translated?')
         ),
 
+        'npi_validate' => array(
```

Comment:
These names appear in PHP global namespace, so I have been prefixing new ones with "gbl_" to bring some order to this.